modules/ROOT/pages/directives/database-mapping.adoc

```diff
@@ -216,6 +158,67 @@ await startStandaloneServer(server, {
 });
 ----
 
+== `@relationship`
+
+Relationships are represented by marking particular fields with a directive -- in this case, `@relationship`. 
+It defines the relationship type in the database, as well as which direction that relationship goes in.
+
+To add a second node type, "Actor", and connect the two together, you should do the following:
+
+[source, graphql, indent=0]
+----
+type Movie @node {
+    title: String
+    actors: [Actor!]! @relationship(type: "ACTED_IN", direction: IN)
+}
+
+type Actor @node {
+    name: String
+    movies: [Movie!]! @relationship(type: "ACTED_IN", direction: OUT)
+}
+----
+
+Note that, in this case, there is a directive on each "end" of the relationship, but it is not essential.
```

Comment:
Let's keep this outside this PR as it is as it is on 6.x, the only reason why it is marked as a changed is because I moved the `@node` description at the top of the page to remark the new importance.